results/2020/education.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Learning & Education
+position: 3
+---
+
+# 2.Education and Learning
+
+There is a lot to be said when talking about the educational system in Morocco. We asked the community about their thoughts, opinion, source of learning and potential solution. Additionally, more than 70% of all respondents confirm that English is no more an obstacle to learning new things
```

Comment:
English is not an obstacle to learning new things anymore.

Reply:
I'm not sure if this is what you are trying to say here, let me know if i didn't understand it